bin/sdc-migrate

```diff
@@ -269,16 +269,14 @@ Usage:
 
 Options:
 
-    -a, --all       List all migrations.
     -l, --long      Long format - includes original and target server uuids.
     -n, --cn        Filter to *only* migrations originating from this CN.
-    -s, --state     Filter on this state (any, running, succcessful, failed).
-                    Defaults to state "active", which is any of running,
-                    paused or failed.
+    -s, --state     Filter on this state (paused, running, succcessful, failed).
 EOF
                 return 1;
                 ;;
 
+            # state=all is deprecated - here for backwards compatibility
             -a|--all)
                 state=all
                 ;;
```

Comment:
Shouldn't we add a deprecation notice here then?

Reply:
Yes - that would be even better.